integration_tests/e2e/goal/review.cy.ts

```diff
@@ -106,7 +108,7 @@ context('Review goal(s)', () => {
     cy.visit(`/plan/${prisonNumber}/view/overview`)
     const overviewPage = Page.verifyOnPage(OverviewPage)
 
-    let createGoalPage = overviewPage.clickAddGoalButton()
+    const createGoalPage = overviewPage.clickAddGoalButton()
```

Comment:
I had to change this to a `const`, and comment the lines out below because of a compilation error (the previous page class was `CreateGoalPage` and the new one is `CreateGoalsPage` (plural), and the methods on those 2 page classes are different at the moment
It doesnt really matter though, because the whole cypress test is disabled, so the whole thing will be deleted at some point soon 👍